libs/game/multiplayer.ts

```diff
@@ -0,0 +1,21 @@
+namespace multiplayer {
+    //% shim=multiplayer::getCurrentImage
+    declare function getCurrentImage(): Image;
+
+    //% shim=multiplayer::setOrigin
+    declare function setOrigin(origin: string): void;
+
+    //% shim=multiplayer::getOrigin
+    declare function getOrigin(): string;
+
+    export function init() {
+        game.addScenePushHandler(() => {
+            game.eventContext().registerFrameHandler(scene.MULTIPLAYER_MESSAGING_PRIORITY, () => {
+                if (getOrigin() === "client") {
+                    const im: Image = getCurrentImage();
+                    scene.setBackgroundImage(im);
+                }
+            });
+        });
+    }
```

Comment:
```suggestion
            game.eventContext().registerFrameHandler(scene.MULTIPLAYER_MESSAGING_PRIORITY, () => {
                if (getOrigin() === "client") {
                    const im: Image = getCurrentImage();
                    scene.setBackgroundImage(im);
                    // clear default menu button behavior
                    controller.menu.onEvent(ControllerButtonEvent.Pressed, () => {});
                }
            });
        });
        game.pushScene();
    }
```

Ah I think this is likely what's causing the issue you mentioned in standup, the scene push handler was running before it was registered when this code lived in pxt-arcade but now we should probably call a push scene to make sure the frame handler registers. (and also added a clear menu so we can't get into a weird state with that)

Reply:
Yay you fixed my bug! I was missing the scene push!